setup.sh

```diff
@@ -69,7 +69,7 @@ if [ -z "$PREFIX" ]; then
 fi
 
 # All 'bpkg' supported commands
-CMDS="json install package term suggest init utils update list show getdeps run source"
+CMDS="json install package term suggest init utils update list show getdeps run source env"
```

Comment:
Minor nitpick: should we order the commands alphabetically? ( :eyes: for easier human-eyes scanning)

```sh
CMDS="env getdeps init install json list package run show source suggest term update utils"
```